templates/admin/article/list_preview.html.twig

```diff
@@ -1,15 +1,13 @@
-{% if object.published %}
-    {% if object.isForRenaissance %}
-        <a href="{{ path('article_view', {categorySlug: object.category.slug, articleSlug: object.slug, app_domain: renaissance_host}) }}" class="btn btn-sm btn-default view_link"
-           title="Prévisualiser" target="_blank" rel="noopener noreferrer">
-            <i class="fa fa-search-plus" aria-hidden="true"></i>
-            Prévisualiser
-        </a>
-    {% else %}
-        <a href="{{ path('article_view', {categorySlug: object.category.slug, articleSlug: object.slug}) }}" class="btn btn-sm btn-default view_link"
-           title="Prévisualiser" target="_blank" rel="noopener noreferrer">
-            <i class="fa fa-search-plus" aria-hidden="true"></i>
-            Prévisualiser
-        </a>
-    {% endif %}
+{% if object.isForRenaissance %}
+    <a href="{{ path('article_view', {categorySlug: object.category.slug, articleSlug: object.slug, app_domain: renaissance_host, preview: article_preview_admin_key}) }}" class="btn btn-sm btn-default view_link"
```

Comment:
preview key should be set only for unpublished articles

Reply:
ok